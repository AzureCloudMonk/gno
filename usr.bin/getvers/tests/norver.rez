 
// Special test case for getvers: no rVersion resource (only a comment)

#include "mytypes.rez"

resource rComment (1, purgeable3) {"This resource fork has no rVersion"};
