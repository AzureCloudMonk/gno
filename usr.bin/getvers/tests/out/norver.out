norver.r:	No version information
